app/services/idv/applicant.rb

```diff
@@ -0,0 +1,34 @@
+module Idv
+  class Applicant
+    attr_reader :profile
+
+    def initialize(applicant, user, password)
+      @profile = build_profile(applicant, user, password)
+    end
+
+    private
+
+    # rubocop:disable MethodLength
+    # This method is single statement spread across many lines for readability
+    def pii_from_applicant(applicant)
+      Pii::Attributes.new_from_hash(
+        first_name: applicant.first_name,
+        middle_name: applicant.middle_name,
+        last_name: applicant.last_name,
+        address1: applicant.address1,
+        address2: applicant.address2,
+        city: applicant.city,
+        state: applicant.state,
+        zipcode: applicant.zipcode,
+        dob: applicant.dob,
+        ssn: applicant.ssn,
+        phone: applicant.phone
+      )
+    end
+    # rubocop:enable MethodLength
+
+    def build_profile(applicant, user, password)
+      Profile.create_with_encrypted_pii(user, pii_from_applicant(applicant), password)
+    end
```

Comment:
`build_profile` implies calling `new` and not saving, but then `create_with_encrypted_pii` is called, which does save to the database. Perhaps this class should be renamed to `ProfileFromApplicant` and the `build_profile` method should be renamed to `create`, with the contents of the `create_with_encrypted_pii`  from the Profile class? This will keep both the Profile class clean and keep the profile creation in one place.